ee/clickhouse/models/property.py

```diff
@@ -13,7 +13,7 @@ def parse_filter(filters: List[Property]) -> Tuple[str, Dict]:
     result = ""
     params = {}
     for idx, prop in enumerate(filters):
-        result += "{cond}(ep.key = %(k{idx})s) AND (ep.value = %(v{idx})s)".format(
+        result += "{cond}(ep.key = %(k{idx})s) AND (trim(BOTH '\"' FROM ep.value) = %(v{idx})s)".format(
```

Comment:
I'd leave this out for now. It's better to add this to the materialized view on clickhouse so that we don't have to clean anything up in post.

Reply:
Hm I think my tests fail if I don't do this. Is it easy to add it to materliazed view?

Reply:
Let's keep this in for now... but I'd like to get that solved at the source as quickly as possible. When we change serialization from JSON to Avro or Proto this will be cleaned up upstream